src/NosCore.GameObject/ComponentEntities/Extensions/CharacterEntityExtension.cs

```diff
@@ -700,5 +701,39 @@ public static CInfoPacket GenerateCInfo(this ICharacterEntity visualEntity)
                 ArenaWinner = false
             };
         }
+
+        public static BptPacket GenerateBpt(this ICharacterEntity _, IClock clock, IOptions<WorldConfiguration> worldConfig)
```

Comment:
this is not linked in anyway to a character so it shouldn't be in this file. Pretty sure it's sent only once so should be directly there